### PR TITLE
[dataset-nav][bug] get connections by cluster and update client namespace

### PR DESCRIPTION
**Pull request by kavilla**
### Description

There was a conflict in namespaces when registering a transport client with OpenSearch cluster. The schemas were also conflicting with observability and data source mgmt endpoints.

Modifying the namespace for query enhancements to avoid conflicts and then initialize the external datasource connections on load.

### Issues Resolved

n/a

## Changelog

- fix: dataset nav to load external connections and update namespace

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/query_enhancements/server/plugin.ts [hunk 1]
```diff
@@ -18,17 +18,16 @@ import { ConfigSchema } from '../common/config';
 import { defineRoutes } from './routes';
 import {
   pplSearchStrategyProvider,
+  pplRawSearchStrategyProvider,
```
Comment (ashwin-pc) [inline src/plugins/query_enhancements/server/plugin.ts:21]: nit: are we using this?

File: src/plugins/query_enhancements/server/search/index.ts [hunk 2]
```diff
@@ -4,5 +4,6 @@
  */
 
 export { pplSearchStrategyProvider } from './ppl_search_strategy';
+export { pplRawSearchStrategyProvider } from './ppl_raw_search_strategy';
```
Comment (ashwin-pc) [inline src/plugins/query_enhancements/server/search/index.ts:7]: nit: I dont see this being referenced anywhere

**Review by ashwin-pc — APPROVED** [review]
LGTM, small nits but none are blockers

**Review by abbyhu2000 — APPROVED** [review]

File: src/plugins/query_enhancements/server/search/index.ts (hunk 2 above)
Comment (kavilla) [reply src/plugins/query_enhancements/server/search/index.ts:7]: it probably wasn't required but i just moved it to the right location it was previously being exported differently than the other search strategies.,

**Pull request merged**